docs/CONTRIBUTING.rst

```diff
@@ -79,27 +79,46 @@ From your local copy directory, use the following commands.
     * Include the relevant issue number if applicable.
     * Before submitting, please ensure that:
       * The proposed changes include an addition to ``CHANGELOG.md`` describing your changes. To label the change with the PR number, you will have to first create the PR, then edit the ``CHANGELOG.md`` with the PR number, and push that change.
-      * The code follows our coding style. This can be checked running ``flake8`` from the source directory.
+      * The code follows our coding style. This can be checked running ``ruff`` from the source directory.
     * **NOTE:** Contributed branches will be removed by the development team after the merge is complete and should, hence, not be used after the pull request is complete.
 
 .. _sec-styleguides:
 
-Styleguides
------------
+Style Guides
+------------
+
+Python Code Style Guide
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Before you create a Pull Request, make sure you are following the Black_ preview style guide, which follows PEP8,
+with code E203 ignored. We also break from the Black format by configuring it with a max line length of 120.
+To check whether your code conforms to the Black_ preview style guide, simply run the ``black`` tool in the project's root
+directory with the ``--check`` argument. You can also run the ``black`` tool without the ``--check`` argument to have
+black automatically format the codebase to comply with the black preview style guide.
 
-Python Code Styleguide
-^^^^^^^^^^^^^^^^^^^^^^
+We also use the ruff_ tool to sort python imports automatically and double-check that the codebase
+conforms to PEP8 standards, and the codespell_ tool to check spelling.
```

Comment:
We also use the ruff_ tool to sort python imports automatically and double-check that the codebase
conforms to PEP8 standards, while using the codespell_ tool to check spelling.

Reply:
FYI you can use the "suggestion" option in a review comment to make suggestions like this. That is more readable, and I would just need to press accept to commit the change.